packages/api/src/post-build.js

```diff
@@ -192,6 +236,30 @@ class PostBuildHandler {
         handleError(res, err);
       }
     });
+
+    this.app.post('/api/build-gh', async (req, res, next) => {
```

Comment:
Will you add a comment above this explaining what this endpoint does, especially that part that is expected to already be done?